keps/sig-storage/2589-csi-migration-portworx/README.md

```diff
@@ -24,50 +72,48 @@ This document present as a vendor specific KEP for the parent KEP
 This inherits all the contents from its parent KEP. It will introduce two new feature gates to be 
 used as described in its parent KEP. For all other contents, please refer to the parent KEP.
 
-### New Feature Gates
+## Motivation
+
+Currently the Portworx volume provisioning happens through Portworx in-tree driver. As part of the parent KEP [CSI Migration](https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/625-csi-migration), the Portworx in-tree driver needs to be deprecated in order to give to use Portworx CSI driver.
+
```

Comment:
Suggest change "the Portworx in-tree driver needs to be deprecated in order to give to use Portworx CSI driver." to the following:
the Portworx in-tree driver logic needs to be "migrated" to use Portworx CSI driver instead.